src/lib/providers.ts

```diff
@@ -0,0 +1,7 @@
+export const providers = {
+    sandbox: 'sandbox',
+    worldpay: 'worldpay',
+    smartpay: 'smartpay',
+    epdq: 'epdq',
```

Comment:
do we still need `epdq`?

Reply:
Hmm, well the value in this enum isn't used so technically no. We still have epdq accounts/payments that are represented in Toolbox, and we could potentially use this global enum in more places, but yeh no need for it.